Automattic/wp-calypso
# SPT: set static preview as default

This PR sets static preview for thumbnails as default.

#### Testing instructions

1) Apply these changes
2) Confirm that the templates thumbnails are images:

<img width="1132" alt="Screen Shot 2019-08-26 at 5 02 57 PM" src="https://user-images.githubusercontent.com/77539/63719520-72fb6780-c823-11e9-820e-75fbe508e46f.png">

See #35791.
Reverted in #34839.
